### PR TITLE
Do not lint 3rd party dependencies. 

### LINKED ISSUE #2: `tuist lint code` reports violations in `Tuist/Dependencies`
**Describe the bug**
When running `tuist lint code`, the command fails because there are violations in the `Tuist/Dependencies` folder.

Tuist version 2.6.0

### PR DESCRIPTION
**Pull request by laxmorek**
Fixes #2

### CODE REVIEW

File: Sources/TuistPluginSwiftLintFramework/Constants/Constants.swift [hunk 1]
```diff
@@ -4,4 +4,13 @@ import Foundation
...
     static let version = "0.0.1"
+    
+    /// A set of constant values that reflects values from Tuist project.
+    enum Tuist {
+        /// A name of the directory that contains Tuist files.
+        static let tuistDirectoryName = "Tuist"
+        
+        /// A name of the directory that contains 3rd party dependencies installed using `Dependencies.swift`.
+        static let dependenciesDirectoryName = "Dependencies"
+    }
```
Comment (danyf90) [inline Sources/TuistPluginSwiftLintFramework/Constants/Constants.swift:15]: Maybe it's worth to define the `Constants` in main tuist repo in a dedicated small module, and just depend on that from here?

File: fixtures/Example1/Tuist/Config.swift [hunk 2]
```diff
@@ -1,5 +1,3 @@
 import ProjectDescription
 
-let config = Config(
-    generationOptions: []
-)
+let config = Config()
```
Comment (danyf90) [inline fixtures/Example1/Tuist/Config.swift:3]: I think you can just remove the whole file

**Review by danyf90 — APPROVED** [review]

File: Sources/TuistPluginSwiftLintFramework/Constants/Constants.swift (hunk 1 above)
Comment (laxmorek) [reply Sources/TuistPluginSwiftLintFramework/Constants/Constants.swift:15]: A target from the `ProjectAutomations` could have `isExternal` property.

Comment (danyf90) [reply Sources/TuistPluginSwiftLintFramework/Constants/Constants.swift:15]: yes, I think it wouold be useful also to have it in TuistGraph.Target, so we don't need to rely on this kind of logic

**Comment by laxmorek** [comment]
Updated.

@danyf90

**Review by danyf90 — APPROVED** [review]

**Pull request merged**